clusterloader2/pkg/measurement/common/service_creation_latency.go

```diff
@@ -50,7 +50,10 @@ const (
 )
 
 func init() {
-	measurement.Register(serviceCreationLatencyName, createServiceCreationLatencyMeasurement)
+	if err := measurement.Register(serviceCreationLatencyName, createServiceCreationLatencyMeasurement); err != nil {
+		klog.Errorf("cant register service %v", err)
```

Comment:
Still not what I wanted - please use Fatalf

https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/pkg/measurement/common/etcd_metrics.go#L40